packages/password/src/index.ts

```diff
@@ -1,7 +1,18 @@
 import input from '@inquirer/input';
 import chalk from 'chalk';
 
-export default (config, ...args) => {
+type PasswordConfig = {
```

Comment:
To answer the other question from your post, I think this should become something like

```ts
type PasswordConfig = InputConfig & { ... };
```

Once the input prompt is migrated, we should export (or extract if possible) its config type and reuse it here since the password prompt is only a superset.

Reply:
Alright will do, would you like me to close this request and make a new one in a day or two?

Reply:
Oh you can just update this branch I think. But up to you